File: app/src/main/java/com/chooloo/www/callmanager/ui/contacts/ContactsMvpView.java
package com.chooloo.www.callmanager.ui.contacts;

import androidx.annotation.Nullable;

import com.chooloo.www.callmanager.entity.Contact;
import com.chooloo.www.callmanager.ui.cursor.CursorMvpView;

public interface ContactsMvpView extends CursorMvpView {

    void load(@Nullable String phoneNumber, @Nullable String contactName);

    void openContact(Contact contact);
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/contacts/ContactsMvpView.kt
package com.chooloo.www.callmanager.ui.contacts

import com.chooloo.www.callmanager.entity.Contact
import com.chooloo.www.callmanager.ui.list.ListMvpView

interface ContactsMvpView : ListMvpView {

    fun observe(): Any?
    fun openContact(contact: Contact)
}